internal/operators/lvm/lvm_operator.go

```diff
@@ -95,6 +91,16 @@ func (o *operator) ValidateCluster(_ context.Context, cluster *common.Cluster) (
 		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetClusterValidationID(), Reasons: []string{message}}, nil
 	}
 
+	minOpenshiftVersionForMultiNodeSupport, err = version.NewVersion(o.config.LvmMinMultiNodeSupportVersion)
+	if err != nil {
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{err.Error()}}, nil
+	}
+	if ocpVersion.LessThan(minOpenshiftVersionForMultiNodeSupport) && !common.IsSingleNodeCluster(cluster) {
+		message := fmt.Sprintf("Logical Volume Manager is only supported for highly available openshift with version %s or above",
+			minOpenshiftVersionForMultiNodeSupport.String())
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetClusterValidationID(), Reasons: []string{message}}, nil
+	}
+
```

Comment:
small nit, 

maybe we should wrap it with if, and check only what is necessary for the case
no need to check the version twice 
```suggestion
	if common.IsSingleNodeCluster(cluster) {
		minOpenshiftVersionForLvm, err = version.NewVersion(o.config.LvmMinOpenshiftVersion)
		if err != nil {
			return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{err.Error()}}, nil
		}
		if ocpVersion.LessThan(minOpenshiftVersionForLvm) {
			message := fmt.Sprintf("Logical Volume Manager is only supported for openshift versions %s and above", o.config.LvmMinOpenshiftVersion)
			return api.ValidationResult{Status: api.Failure, ValidationId: o.GetClusterValidationID(), Reasons: []string{message}}, nil
		}
	} else {
		minOpenshiftVersionForMultiNodeSupport, err = version.NewVersion(o.config.LvmMinMultiNodeSupportVersion)
		if err != nil {
			return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{err.Error()}}, nil
		}
		message := fmt.Sprintf("Logical Volume Manager is only supported for highly available openshift with version %s or above",
			minOpenshiftVersionForMultiNodeSupport.String())
		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetClusterValidationID(), Reasons: []string{message}}, nil
	}
```

Reply:
I believe this logic works because minOpenshiftVersionForMultiNodeSupport is always greater than minOpenshiftVersionForLvm, I wrote the logic without this assumption.

Good catch, will correct.